src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1727,11 +1727,12 @@ def get_timeseries_multiple_flows(settings, dict_asset, file_name, header):
 
 
 def process_maximum_cap_constraint(dict_values, group, asset, subasset=None):
+    # ToDo: should function be split into separate processing and validation functions?
```

Comment:
I tried to check if all lines of this function are covered in the pytests, but I am not sure actually. @Bachibouzouk posted a command somewhere which showed us how to execute the pytests and coverage following after that. You could do that for the `C0` module and see if everything tested. If it is, nothing has to be done - if it is not, there might be room for improvement, and it may be nice to create smaller functions to ease testing.

Sadly, online in coveralls, only the `dev` branch is evaluated.

Reply:
This is the command: `EXECUTE_TESTS_ON=master coverage run -m pytest`. I guess you can also specify a file...

Reply:
@smartie2076 I will check that then, but maybe this is an issue for #835? Let me know what you think about that

Reply:
Sure, it is not needed. Just wanted to give you the tools to decide whether or not you wanted to split into multiple functions. In general, small functions are better to test, but if your tests already cover everything, I´d say that splitting is not necessary now.

Reply:
I'll remember this for when I do the benchmark test, thanks :)